ktor-network/nix/src/io/ktor/network/selector/SelectUtils.kt

```diff
@@ -89,8 +90,10 @@ internal class SelectorHelper {
                 continue
             }
 
-            pselectBridge(maxDescriptor + 1, readSet.ptr, writeSet.ptr, errorSet.ptr)
-                .check()
+            try {
+                pselectBridge(maxDescriptor + 1, readSet.ptr, writeSet.ptr, errorSet.ptr).check()
+            } catch (_: PosixException.BadFileDescriptorException) {
+            }
```

Comment:
please add a comment why it is ok to ignore this exception

Reply:
Sure, will add